features/polyExplorer/src/components/clusterStories/baseClusterStory/baseClusterStory.jsx

```diff
@@ -121,10 +121,10 @@ const DataStory = ({
                 <div className="scrollytelling-marks" ref={marksContainer}>
                     {buildMarks()}
                 </div>
-                <div className="story-sections">{children}</div>
+                {children}
             </div>
         </Screen>
     );
 };
 
-export default DataStory;
+export default ClusterStory;
```

Comment:
If you put this in line, you will probably save changing names if there's any other change. If it's a export default, it should probably go with what's exported anyway.